packages/wdio-browserstack-service/src/util.ts

```diff
@@ -566,7 +566,7 @@ export function getCiInfo () {
         }
     }
     // CircleCI
-    if (env.CI === 'true' && env.CIRCLECI === 'true') {
+    if (isTrue(env.CI) && isTrue(env.CIRCLECI)) {
```

Comment:
let's not do this as for some comparisons something else than 'true' is used and the change is not required for the PR for it to work.

Reply:
I'm not sure I understand. Can you give an example where the earlier change works but this doesn't?

Reply:
The code is working fine, that wasn't my point. Looking back at it, I don't really mind the change so I'll resolve this :)